src/Components/SignUpPage.jsx

```diff
@@ -0,0 +1,147 @@
+            </div>
+
+            {!nameError && (
+              <p className="signUp__name-input-error">
+                {" "}
+                Please enter name inside input
+              </p>
+            )}
+          </div>
+          <div className="signUp__email-input-box">
+            <div className="signUp__email-input-inner-box">
+              <input
+                className="signUp__email-input"
+                type="email"
+                name="email"
+                placeholder="Email"
+                required
+              />
+            </div>
+            {!emailError && (
+              <p className="signUp__email-input-error">
+                Please check your email !
+              </p>
+            )}
+          </div>
+          <div className="signUp__password-input-box">
+            <div className="signUp__password-input-show-btn-inner-box">
+              <input
+                className="signUp__password-input"
+                type={showPassword ? "text" : "password"}
+                name="password"
+                placeholder="Password"
+                required
+              />
+              <button
+                className="signUp__show-password-btn"
+                onClick={togglePassword}
+                type="button"
+              >
+                {showPassword ? (
+                  <svg
+                    xmlns="http://www.w3.org/2000/svg"
+                    width="24"
+                    height="24"
+                    viewBox="0 0 24 24"
+                    fill="none"
+                    stroke="currentColor"
+                    stroke-width="2"
+                    stroke-linecap="round"
+                    stroke-linejoin="round"
+                    class="lucide lucide-eye-off"
+                  >
+                    <path d="M10.733 5.076a10.744 10.744 0 0 1 11.205 6.575 1 1 0 0 1 0 .696 10.747 10.747 0 0 1-1.444 2.49" />
+                    <path d="M14.084 14.158a3 3 0 0 1-4.242-4.242" />
+                    <path d="M17.479 17.499a10.75 10.75 0 0 1-15.417-5.151 1 1 0 0 1 0-.696 10.75 10.75 0 0 1 4.446-5.143" />
+                    <path d="m2 2 20 20" />
+                  </svg>
+                ) : (
+                  <svg
+                    xmlns="http://www.w3.org/2000/svg"
+                    width="24"
+                    height="24"
+                    viewBox="0 0 24 24"
+                    fill="none"
+                    stroke="currentColor"
+                    stroke-width="2"
+                    stroke-linecap="round"
+                    stroke-linejoin="round"
+                    class="lucide lucide-eye"
+                  >
+                    <path d="M2.062 12.348a1 1 0 0 1 0-.696 10.75 10.75 0 0 1 19.876 0 1 1 0 0 1 0 .696 10.75 10.75 0 0 1-19.876 0" />
+                    <circle cx="12" cy="12" r="3" />
+                  </svg>
+                )}
+              </button>
+            </div>
+            {!passwordError && (
```

Comment:
after changing the above code in the function this should be without ! mark then